locale/ca/locale.po

```diff
@@ -1425,10 +1425,6 @@ msgstr "Avisos públics"
 
 msgid "notification.type.editorAssignmentTask"
 msgstr "S'ha enviat una nova monografia a la qual s'ha d'assignar un editor/a."
-
```

Comment:
Locale entries need  a space between them or the `msgid`/`msgstr` pair won't properly work.